src/status_im/config.cljs

```diff
@@ -72,7 +72,7 @@
 
 ;; CONFIG VALUES
 (def log-level (string/upper-case (get-config :LOG_LEVEL "")))
-(def fleet (get-config :FLEET "waku.sandbox"))
+(def fleet (get-config :FLEET ""))
```

Comment:
so there is no default value if FLEET is not set in config file?

Reply:
The value is set in status-go as the default fleet if empty, if you override it in the client, it will become "sticky", meaning that it will always be that one unless you change it in the settings, even if we update the default fleet (i.e the same as if you would change it in the settings). So it should only be used for testing, most users will just want to use the default fleet and leave it blank